src/lib.js

```diff
@@ -63,6 +104,13 @@ var curlConverter = {
             }
         }
 
+        // cannot send HEAD request in the curl command with POST data option and without --get/-G
+        // Ex- 'curl -I http://example.com -d "a=b"' will throw an error.
+        if ((curlObj.data.length > 0 || curlObj.dataAscii.length > 0 || curlObj.dataBinary || curlObj.dataUrlencode.length > 0) && 
```

Comment:
@Dhroov7 `curlObj.data.length > 0 || curlObj.dataAscii.length > 0 || curlObj.dataBinary || curlObj.dataUrlencode.length` can be moved to a function